crates/driver/src/infra/solver/dto/auction.rs

```diff
@@ -119,13 +119,14 @@ impl Auction {
                             competition::order::Kind::Limit { .. } => Class::Limit,
                             competition::order::Kind::Liquidity => Class::Liquidity,
                         },
-                        fee_policies: order
-                            .protocol_fees
-                            .iter()
-                            .filter(|_| fee_handler == FeeHandler::Solver)
-                            .cloned()
-                            .map(Into::into)
-                            .collect(),
+                        fee_policies: None,
+                        // order
```

Comment:
I left it here on purpose to fully place it back once we can do this change in the API.

Reply:
hmh, I think we already need to purposefully turn it on for solvers, in that case they should be expecting this field. Will commandeer